### DISCUSSION TITLE
Can't Install on Single Board - Resolved

### DISCUSSION THREAD

**Discussion by mbhforum** [tesla-android/issue-tracker#90] [Q&A] [answered]
Tried install instructions for 2022.38.1.  During the install, after I enter the below command:

![image](https://user-images.githubusercontent.com/11696289/193125034-1fe16ff3-4f95-48f3-8302-2a302663dba8.png)

My Mac loses connectivity with the RPi and I am no longer to run the next command. I had no issues with 2 RPi's installing.  I am using a new Amazon Basics USB-C to USB-A that is connected to my Mac, but I am able to run the first set of commands fine.

**Comment by mbhforum** [comment, marked as answer]
I fixed this issue.  I think it was because I had the HDMI Capture card and CarLink plugged in during setup and Android was complaining about USB devices

**Reply by BroRidersMTL** [reply]
I had same issue but i had a bad hdmi-cs2 card
